File: python/7 kyu/290.py
# https://www.codewars.com/kata/5656b6906de340bd1b0000ac/train/python

# Passed

def longest(a1, a2):
    all_unique_letters = set(a1).union(set(a2))
    sorted_letters = sorted(list(all_unique_letters))
    result = "".join(sorted_letters)
    
    return result

output = longest("loopingisfunbutdangerous", "lessdangerousthancoding")
print(output)

File: nodejs/7 kyu/290.js
// https://www.codewars.com/kata/57ed30dde7728215300005fa/train/javascript

// Passed

function bump(x) {
  let bumpsCount = 0;
  let result = "Woohoo!";
  for (const roadPart of x) {
    if (roadPart === "n") {
      bumpsCount++;
    }

    if (bumpsCount > 15) {
      result = "Car Dead";
      break;
    }
  }
  return result;
}

const output = bump("__nn_nnnn__n_n___n____nn__nnn");
console.log(output);